deploy/Chart/templates/dashboard/serviceaccount.yaml

```
@@ -0,0 +1,10 @@
+{{- if .Values.dashboard.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dashboard-account
```

Please make it consistent with the others by removing suffix.

```suggestion
  name: dashboard
```